vividus-allure-adaptor/src/main/java/org/vividus/bdd/report/allure/AllureReportGenerator.java

```diff
@@ -111,16 +115,29 @@ private void generateReport()
         LOGGER.info("Allure report is successfully generated at: {}", reportDirectory.getAbsolutePath());
     }
 
-    private void writeExecutorInfo() throws IOException
+    private void writeCategoriesInfo() throws IOException
```

Comment:
I believe you need to add validations to unit tests to be sure, that `categories.json` is generated